stage2.1_traited_script/traited_face_detect.py

```diff
@@ -0,0 +1,90 @@
+""" Script analyzing an image, detecting human faces inside it, and printing
+EXIF data about it.
+"""
+import PIL.Image
+from PIL.ExifTags import TAGS
+from skimage import data
+from skimage.feature import Cascade
+import matplotlib.pyplot as plt
+from matplotlib import patches
+from os.path import join
+import numpy as np
+
+# ETS imports
+from traits.api import Dict, File, HasStrictTraits, List, observe
+
+
+class ImageFile(HasStrictTraits):
```

Comment:
I am not sure why we are introducing `HasStrictTraits` everywhere? This seems fine eventually but I am really not sure why it is needed here for a beginner.

Reply:
Discussed offline.